bindings/bindingtester/known_testers.py

```diff
@@ -47,10 +47,33 @@ def get_test(cls, test_name_or_args):
             return Tester(test_name_or_args.split(' ')[0], test_name_or_args)
 
 
+    # get_cmd():
+    #   return the command associated with the tester
+    #   It augments the basic command with cbatrace, i.e., C binding Api Tracing, 
+    #   command if needed. 
+    # NOTE: 
+    #   - Set environment variable "CBATRACE_ENABLE" to 1 to enable cbatrace 
+    #   - For now, cba tracing is only enabled for java tests
+    def get_cmd(self):
+      # check whether cba tracing is enabled  
+      if (os.environ.get('CBATRACE_ENABLE') == '1' and self.name.startswith('java')):
```

Comment:
Is it feasible to make these parameters of the binding tester script rather than environment variables? This would be more similar to other flags we can set on the tester.

Also, changes here in known testers won't be applied to any tester run manually, which does happen from time to time.